### DISCUSSION TITLE
Ability to pass a prefix for env variables

### DISCUSSION THREAD

**Discussion by jomakori** [kienstra/localci#172] [Ideas]
In our current setup, we use Dopper cli to pass in environmental variables. I manually added the `doppler run -- prefix` to the command passed to get my environment vars passed in.

Ex) 
```
doppler run -- /home/ec2-user/.vscode-server/extensions/localci.local-ci-1.7.9/bin/circleci/Linux/x64/bin/circleci local execute --job 'check-updated-files' --config /tmp/local-ci/devops/process.yml -v /tmp/local-ci/devops/volume:/tmp/local-ci
```

**Comment by kienstra** [comment]
That's a really good idea, the lack of env vars is holding Local CI back.

It sounds like you'd like to be able to do that via the VS Code UI, not have to run that command manually.

**Reply by jomakori** [reply]
Yeah, it'll help make it a cleaner run

**Reply by kienstra** [reply]
Sure, that'd make it much more useful.

Would you like some kind of configuration in your VS Code `settings.json`, like:

```json
    "localCI.useDoppler": true,
```

**Comment by kienstra** [comment]
Implemented in https://github.com/getlocalci/local-ci/pull/237
